engines/cge2/vmenu.cpp

```diff
@@ -25,16 +25,22 @@
  * Copyright (c) 1994-1997 Janusz B. Wisniewski and L.K. Avalon
  */
 
+#include "common/config-manager.h"
+#include "common/text-to-speech.h"
 #include "cge2/text.h"
 #include "cge2/vmenu.h"
 #include "cge2/events.h"
 
 namespace CGE2 {
 
+const char *_iWantToStopPlayingAlready;
```

Comment:
these aren't initialized - so they might send garbage in `VMenu::touch` to `ttsMan->say`